05-script.md

```diff
@@ -440,3 +440,34 @@ Of course, this introduces another tradeoff between flexibility and complexity.
 > # Script 3
 > echo $@.dat
 > ~~~
+
+> ## Ooops my large script has a bug. How can I easily find the problem? {.challenge}
+>
+> You have written a large script with many lines, and it's not working properly.
+> How can I easily identify the problem ?
```

Comment:
A minor comment here: this uses "I" and "you" a bit inconsistently: "you" have written the script but "I" identify the problem. I think it is perfectly fine in the title, but here I'd stay with "you". Maybe also consider rephrasing it to "What can you do to help you identify the problem?" -- I am not sure interpreting the output with `bash -x` is necessarily "easy" for the novice learner.